hiring/README.md

```diff
@@ -4,40 +4,120 @@ We believe that our hiring process should be [fair and repeatable](rationale.md)
+ - Discussion of previous experience (20 minutes)
+ - Pairing session (30 minutes)
+ - Problem solving (20 minutes)
+ - Questions (10 minutes)
 
-We then follow up with some more technical, deeper thinking questions documented in this [spreadsheet](https://docs.google.com/spreadsheets/d/1gzOeu_a9uPdOA0roY1oMo_S4pLQCCdrysgyFANcu0FA/edit#gid=1588607612). (Sorry if you're a candidate looking at this, we do not hand out these questions in advance.)
+#### Further introduction to Made
+
+We usually ask the candidate if they have any more questions about what we do, and what the role looks like. It also helps break the ice of the interview conditions.
+
+#### Discussion of previous experience
+
+We then ask the candidate to explain how they got to where they are now. We dig into examples of challenges, what their process looked like and how that differs from their ideal process.
+
+We are particularly looking for strong indications that they:
+
+ - can meet our expectations for the role
+ - can meet our expectations for any traits we see them immediately tackling
+ - can grow with us
+ - are proactive and problem solvers
+ - take responsibility for past failings rather than blame others
+
+#### Pairing session
+
+After this introduction we usually pair with a candidate on a problem for 30 minutes. This step is used to see how well a candidate communicates and works with us. Of course it is a pressured environment, however we try and make the candidate at ease by sharing some of the work and discussing the problem with them.
+
+Our current Engineer pairing task can be found in this [google doc](https://docs.google.com/a/madebymade.co.uk/document/d/1nqKNcy9wGcQjAOD-nYP_KtXLFyDQD6KEEhGw6_Andik/edit?usp=sharing). (Sorry if you're a candidate looking at this, we do not hand out these questions in advance.)
+
+#### Problem solving
+
+We follow up with some more technical, deeper thinking questions documented in this [spreadsheet](https://docs.google.com/spreadsheets/d/1gzOeu_a9uPdOA0roY1oMo_S4pLQCCdrysgyFANcu0FA/edit#gid=1588607612). (Sorry if you're a candidate looking at this, we do not hand out these questions in advance.)
+
+#### Questions
 
 Finally we answer any questions the candidate may have.
 
+### Decide
+
+We will hire a candidate if all interviewers answer yes to all of the following questions:
+
+ - Do we have a strong indication based on their past experience that they will be able to fulfil our expectations for their role and traits?
+ - Do we have a strong indication based on their pairing session that they will be able to fulfil our expectations for their role and traits?
+ - Do we have a strong indication based on their problem solving that they will be able to fulfil our expecations for their role and traits?
+
 ### Hire
 
 Once we've determined that a candidate is going to be a good hire for us and the candidate thinks we're a good fit for their next role, we hire them. We will prepare a contract and send it to them as soon as possible.
```

Comment:
Once we've determined that a candidate is going to be a good hire for us and the candidate thinks we're a good fit for their next role, we will _offer them the position_.

?